### PR TITLE
chore(apex sink): remove the sink

### PR DESCRIPTION
**Pull request by neuronull**
The service is no longer available as of 12/31/2022.

closes: #16338

### CODE REVIEW

File: .github/semantic.yml [hunk 1]
```diff
@@ -185,7 +185,6 @@ scopes:
 
   # sinks
   - amqp sink # Anything `amqp` sink related
-  - apex sink # Anything `apex` sink related
```
Comment (spencergilbert) [inline .github/semantic.yml:188]: Needs to be removed in the TF resources

**Review by spencergilbert — CHANGES_REQUESTED** [review]
Can we leave a note in the upgrade doc?

File: .github/semantic.yml (hunk 1 above)
Comment (neuronull) [reply .github/semantic.yml:188]: :thinkes: TF resources?

**Comment by neuronull** [comment]
> Can we leave a note in the upgrade doc?

ah yeah I meant to add that 👍

File: .github/semantic.yml (hunk 1 above)
Comment (spencergilbert) [reply .github/semantic.yml:188]: https://github.com/vectordotdev/resources/blob/master/vector/github/semantic_pull_requests.tf

Comment (neuronull) [reply .github/semantic.yml:188]: That file indicates it is autogenerated ...

Comment (spencergilbert) [reply .github/semantic.yml:188]: The file it creates says it's autogenerated and to go to that link to update it

Comment (neuronull) [reply .github/semantic.yml:188]: https://github.com/vectordotdev/resources/pull/192

**Review by spencergilbert — APPROVED** [review]

**Comment by neuronull** [comment]
Hmm I wonder if we need to keep some of the cue after all? The Netfly failed:

```
1:02:53 PM: releases/0.24.0/index.html
1:02:53 PM:   target does not exist --- releases/0.24.0/index.html --> /docs/reference/configuration/sinks/apex/
1:02:54 PM: ========================================================================
1:02:54 PM: ✘✘✘ failed in 733.920183ms
1:02:54 PM: 1 errors in 440 documents
1:02:54 PM: make: *** [Makefile:60: run-link-checker] Error 1
```

**Pull request merged**